front/app/components/Form/Components/Controls/LinearScaleControl.tsx

```diff
@@ -36,20 +35,98 @@ const LinearScaleControl = ({
 }: ControlProps) => {
   const isSmallerThanPhone = useBreakpoint('phone');
   const theme = useTheme();
-  const maximum = schema?.maximum;
+  const { formatMessage } = useIntl();
+
+  const minimum = 1;
+  const maximum = schema?.maximum ?? 7; // Seven since the maximum number of options is 7
+  const totalOptions = maximum;
   const answerNotPublic = uischema.options?.answer_visible_to === 'admins';
```

Comment:
Do we need this separate totalOptions variable? Or could we just use "maximum" instead? :)